charts/cloudzero-agent/README.md

```diff
@@ -121,29 +121,26 @@ Please see the [sizing guide](./docs/sizing-guide.md) in the docs directory.
 
 ### Metric Exporters
 
-This chart depends on metrics from [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) and [node-exporter](https://github.com/prometheus/node_exporter) projects as subcharts.
+This chart depends on metrics from [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) as a subchart.
 
-By default, these subcharts are disabled to allow scraping from existing instances. Configure the `cloudzero-agent` to use existing service endpoint addresses in `values.yaml`:
+By default, this subchart is enabled to allow scraping from existing instances. Configure the `cloudzero-agent` to use existing service endpoint addresses in `values.yaml`:
 
 ```yaml
 validator:
   serviceEndpoints:
      kubeStateMetrics: <kube-state-metrics>.<example-namespace>.svc.cluster.local:8080
-     prometheusNodeExporter: <node-exporter>.<example-namespace>.svc.cluster.local:9100
 ```
```

Comment:
I don't believe this is necessary either with the new validator

Reply:
sounds good, removed

Reply:
it actually looks like the validator container still expects the KSM to be passed in, though I'm not sure why

Reply:
@josephbarnett @bdrennz we can keep this and merge as is, but I think we'd want to remove this requirement from the validator, right? it shouldn't require a KSM endpoint to start given that it is finding it dynamically

Reply:
it looks like the validator still uses that given KMS endpoint for the validation?